Playbooks/Dynamic-Summaries-API-Upsert/azuredeploy.json

```diff
@@ -76,13 +76,13 @@
         }
     },
   "variables": {
+    "AzuremonitorlogsConnectionName": "[concat('Azuremonitorlogs-', parameters('PlaybookName'))]",
     "AzureADConnectionName": "[concat('azuread-', parameters('PlaybookName'))]",
-    "connections_azuremonitorlogs_externalid": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', 'nbauto', '/providers/Microsoft.Web/connections/azuremonitorlogs')]",
+    "connections_azuremonitorlogs_externalid": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Web/connections/', variables('AzuremonitorlogsConnectionName'))]",
     "SubscriptionId": "[concat(subscription().subscriptionId)]",
     "TenantId": "[concat(subscription().tenantId)]",
     "IntegrationAccount_Id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Logic/integrationAccounts/', parameters('IntegrationAccountName'))]",
-    "DynamicSummariesAPIUri": "[concat('https://management.azure.com/subscriptions/', subscription().subscriptionId, '/resourceGroups/', parameters('ResourceGroupNameforDynamicSummaries'), '/providers/Microsoft.OperationalInsights/workspaces/', parameters('WorkspaceNameforDynamicSummaries'), '/providers/Microsoft.SecurityInsights/dynamicSummaries/', parameters('DynamicSummaryGUID'), '?api-version=2023-03-01-preview')]",
-    "AzuremonitorlogsConnectionName": "[concat('Azuremonitorlogs-', parameters('PlaybookName'))]"
+    "DynamicSummariesAPIUri": "[concat('https://management.azure.com/subscriptions/', subscription().subscriptionId, '/resourceGroups/', parameters('ResourceGroupNameforDynamicSummaries'), '/providers/Microsoft.OperationalInsights/workspaces/', parameters('WorkspaceNameforDynamicSummaries'), '/providers/Microsoft.SecurityInsights/dynamicSummaries/', parameters('DynamicSummaryGUID'), '?api-version=2023-03-01-preview')]"
```

Comment:
Instead of taking DynamicSummaryGUID as input parameter, it can be generated by guid function.
https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-string#guid

Reply:
Understood.  This is by design.  We want to keep the same summary for each schedules run.  Otherwise, each run will generate a new summary.

Reply:
guid function is an ARM template function. GUID will be generated during the deployment of ARM template and will remain constant throughout lifetime of playbook. It will not change between playbook runs.

Reply:
done